consul_exporter.go

```diff
@@ -60,6 +60,11 @@ var (
 		"How many members are in the cluster.",
 		nil, nil,
 	)
+	memberStatus = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "", "members_status"),
```

Comment:
Maybe `serf_lan_member_status` to be consistent with the previous metric (eg `consul_serf_lan_members`).